app/grants/templates/grants/card/clr_match.html

```diff
@@ -2,21 +2,13 @@
 
 <div class="clr">
   <h2 class="label font-caption font-weight-semibold mb-1 p-1 text-center">
-    {% if is_round_5_5 %}
-    CLR MATCH ROUND 5.5
```

Comment:
I haven't yet deployed the round 5.5 data stuff but I'm thinking I may not actually need to (will just get it merged into master for posterity) - I'm thinking I'll just manually update the payout numbers from the calcs I ran locally w/ the prod contribution data + givingblock data